helm/prometheus-meta-operator/templates/recording-rules/service-levels.rules.yml

```diff
@@ -36,57 +41,76 @@ spec:
         class: MEDIUM
         service: kubelet
       record: raw_slo_errors
-    - expr: sum(raw_slo_requests) by (service, cluster_type, cluster_id)
-      record: slo_requests
-    - expr: sum(raw_slo_errors) by (service, cluster_type, cluster_id)
-      record: slo_errors
-    - expr: sum(sum_over_time(raw_slo_errors[5m])) by (cluster_type, cluster_id, service, class) / sum(sum_over_time(raw_slo_requests[5m])) by (cluster_type, cluster_id, service, class)
-      record: slo_errors_per_request:ratio_rate5m
-    - expr: sum(sum_over_time(raw_slo_errors[30m])) by (cluster_type, cluster_id, service, class) / sum(sum_over_time(raw_slo_requests[30m])) by (cluster_type, cluster_id, service, class)
-      record: slo_errors_per_request:ratio_rate30m
-    - expr: sum(sum_over_time(raw_slo_errors[1h])) by (cluster_type, cluster_id, service, class) / sum(sum_over_time(raw_slo_requests[1h])) by (cluster_type, cluster_id, service, class)
-      record: slo_errors_per_request:ratio_rate1h
-    - expr: sum(sum_over_time(raw_slo_errors[6h])) by (cluster_type, cluster_id, service, class) / sum(sum_over_time(raw_slo_requests[6h])) by (cluster_type, cluster_id, service, class)
-      record: slo_errors_per_request:ratio_rate6h
```

Comment:
these recording rules moved downwards for some reason

Reply:
Are you moving this because these recordings should run later than other rules?
https://github.com/giantswarm/prometheus-meta-operator/pull/593/files#diff-f2635a7abec77c51bb996d248798f925b42a22eb713a28b021e65785042488cfR65

Reply:
True, that makes sense!